### ISSUE TITLE
Annotation fetching breaks with unhashable type: 'SortableTableWidgetItem'

### ISSUE THREAD

**Issue by Manishearth** [davidfor/calibre-annotations#12]
Plugin version: 1.15
Calibre version: 5.16.1
OS: 64 bit Windows and Linux
Language: English

If I fetch annotations from my Kindle, it is able to collect the relevant annotations just fine (and shows them in a list), but if I click "Import Annotations", it starts importing and at around 90% it shows the following error:

```
calibre, version 5.16.1
Error fetching annotations: <p>Unable to fetch annotations from Amazon Kindle.</p>

Traceback (most recent call last):
  File "calibre_plugins.annotations.action", line 324, in fetch_device_annotations
  File "calibre_plugins.annotations.action", line 1080, in process_selected_books
  File "calibre_plugins.annotations.action", line 503, in generate_confidence
TypeError: unhashable type: 'SortableTableWidgetItem'
```

**Comment by Manishearth** [comment]
Thanks! How much time do you think it'll take for a full release? I don't mind installing a beta but would prefer to use the regular plugin route if at all possible.

**Comment by davidfor** [comment]
Just released 1.16.0 with the fix. Sorry for the delay, I forgot I hadn't done it.
